packages/nextjs/components/CheckinForm.tsx

```diff
@@ -76,66 +77,54 @@ const CheckinForm = ({ latLng = [0, 0] }) => {
 
   return (
     <>
-      <h1>Checkin form view</h1>
       <div className="flex items-center flex-col w-full flex-grow">
         <div className="flex-grow center w-full">
-          <form onSubmit={handleSubmit} className="card m-10 bg-base-100 shadow-xl">
-            <div className="card-body mt-5">
-              <h2 className="card-title">Check-in details:</h2>
-              <div className="flex-col card-actions justify-end">
-                {/* <label className="flex center flex-row items-center w-full">
-                  <div>Address:</div>
-                  <input type="text" name="subject" value={connectedAddress} className="input w-full" />
-                </label> */}
-                <label className="flex center flex-row items-center">
-                  <div>Coordinate:&nbsp;</div>
-                  (
-                  <input
-                    type="number"
-                    name="coordinateInputX"
-                    // placeholder={formValues.coordinateInputX.toString()}
-                    className="input w-full max-w-xs"
-                    value={latLng[0]}
-                    onChange={handleChange}
-                  />
-                  ,
-                  <input
-                    type="number"
-                    name="coordinateInputY"
-                    // placeholder={formValues.coordinateInputY.toString()}
-                    className="input w-full max-w-xs"
-                    value={latLng[1]}
-                    onChange={handleChange}
-                  />
-                  )
-                </label>
-                <label className="flex center flex-row items-center">
-                  <div>Timestamp:</div>
-                  {/* Not Sure we input this, might be on EAS.sol in the case of onchain attestations */}
-                  <input
-                    type="number"
-                    name="timestamp"
-                    value={formValues.timestamp}
-                    onChange={handleChange}
-                    className="input w-full max-w-xs"
-                  />
-                </label>
-                <label className="flex center flex-row items-center w-full">
-                  <div>Data:</div>
-                  {/* Not Sure we input this, might be on EAS.sol in the case of onchain attestations */}
-                  <input
-                    type="text"
-                    name="data"
-                    value={formValues.data}
-                    placeholder={formValues.data}
-                    className="input w-full"
-                    onChange={handleChange}
-                  />
-                </label>
+          <form onSubmit={handleSubmit} className="card m-5 flex-col gap-2 ">
```

Comment:
I have some fine-grained UI feedback here, but we can review that separately, I won't document it in detail here.